CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- New JMX open method with optional parameters. Added remote JMX URL protocol option.
+Use `jmx.OpenWithParameters` instead of old removed methods `jmx.OpenWithSSL` and `jmx.Open` 
```

Comment:
I understand because I checked the code, but users would like some explanation on this.

Reply:
ok